<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShoppingCart</class>
 <widget class="QMainWindow" name="ShoppingCart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1420</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ShoppingCart</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Image.qrc">
    <normaloff>:/Image/Gucci_logo.webp</normaloff>:/Image/Gucci_logo.webp</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>770</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QFrame" name="frame">
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>90</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>70</width>
            <height>70</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/Image/Gucci_logo.webp);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>88</x>
            <y>20</y>
            <width>275</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>18</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border-left: 5px solid rgb(149, 149, 149);
padding-left:1px;</string>
          </property>
          <property name="text">
           <string>SHOPPING CART</string>
          </property>
         </widget>
        </widget>
       </item>
       <item alignment="Qt::AlignRight">
        <widget class="QFrame" name="frame_3">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>55</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QPushButton" name="checkBtn">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>5</y>
            <width>180</width>
            <height>50</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>18</pointsize>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton#checkBtn{
	background-color: rgb(139, 139, 139);
	border-radius:20px;
	border: 2px solid rgba(157,150,153,0.95);
}
QPushButton#checkBtn:hover{
	border-radius:20px;	
	background-color:rgba(102, 94, 95, 0.21);
	border: 3px solid rgba(157,150,153,0.95);
}</string>
          </property>
          <property name="text">
           <string>CHECK</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QTableWidget" name="productTableWidget">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>10</y>
            <width>811</width>
            <height>551</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>15</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border-color: rgb(171, 171, 171);
border-left-color: rgb(182, 182, 182);

</string>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>200</number>
          </attribute>
          <column>
           <property name="text">
            <string>PRODUCT NAME</string>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <bold>true</bold>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>QUANTITY</string>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <bold>true</bold>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>PRICE</string>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <bold>true</bold>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>TOTAL</string>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <bold>true</bold>
            </font>
           </property>
          </column>
         </widget>
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>540</y>
            <width>83</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton#pushButton{
	background-color: rgb(139, 139, 139);
	border-radius:20px;
	border: 2px solid rgba(157,150,153,0.95);
}
QPushButton#pushButton:hover{
	border-radius:20px;	
	background-color:rgba(102, 94, 95, 0.21);
	border: 3px solid rgba(157,150,153,0.95);
}</string>
          </property>
          <property name="text">
           <string>Back</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>770</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-left: 5px solid rgb(149, 149, 149);
padding-left:1px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="summaryFrame">
      <property name="maximumSize">
       <size>
        <width>470</width>
        <height>770</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QFrame" name="frame_9">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QFrame" name="frame_7">
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>70</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="summaryTitle">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border-bottom: 2px solid rgb(149, 149, 149);
padding-left:5px;</string>
            </property>
            <property name="text">
             <string>ORDER SUMMARY</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QFrame" name="frame_5">
         <property name="minimumSize">
          <size>
           <width>450</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>450</width>
           <height>80</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>TOTAL QUANTITY</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignRight">
           <widget class="QLabel" name="quantitySummary">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>56</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_6">
         <property name="minimumSize">
          <size>
           <width>450</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>450</width>
           <height>80</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <pointsize>15</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>TOTAL PRICE</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignRight">
           <widget class="QLabel" name="priceSummary">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>56</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>56</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">#priceSummary{
margin-left:15px;
}</string>
            </property>
            <property name="text">
             <string>$0</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_8">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="checkOutBtn">
            <property name="minimumSize">
             <size>
              <width>180</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>180</width>
              <height>60</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#checkOutBtn{
	background-color: rgb(139, 139, 139);
	border-radius:20px;
	border: 2px solid rgba(157,150,153,0.95);
}
QPushButton#checkOutBtn:hover{
	border-radius:20px;	
	background-color:rgba(102, 94, 95, 0.21);
	border: 3px solid rgba(157,150,153,0.95);
}</string>
            </property>
            <property name="text">
             <string>CHECK OUT</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Image.qrc"/>
 </resources>
 <connections/>
</ui>
